src/controllers/controlleroutputmappingtablemodel.cpp

```diff
@@ -189,6 +190,53 @@ QVariant ControllerOutputMappingTableModel::data(const QModelIndex& index,
     return QVariant();
 }
 
+QString ControllerOutputMappingTableModel::displayString(const QModelIndex& index) const {
+    if (!m_pMidiMapping || !m_pTableView || !index.isValid()) {
+        return QString();
+    }
+
+    int row = index.row();
+    int column = index.column();
+    const MidiOutputMapping& mapping = m_midiOutputMappings.at(row);
+
+    switch (column) {
```

Comment:
Can this be replaces by `data(index, Qt::DisplayRole).toString()`?

Reply:
That's what I tried first, but the display text is not stored in the model, it's obviously generated on the fly when the table's viewport is updated (show, scroll, confirm edits etc.). IIUC that's also why the "Action" column is sorting by raw ConfigKey, not the translated description string.

Reply:
OK